### PR TITLE
docs: add office hour meeting minutes 2023-08-25

### PR DESCRIPTION
**Pull request by carslen**
add recent office hour.

### CODE REVIEW

File: blog/2023-08-25-OfficeHours.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - Contribute to [e2e Helm Chart](https://github.com/eclipse-tractusx/e2e-testing)
+
+### Security
+
+- License for Invicty. Please contact Security Team (Lokesh Gujre or Piotr Stys) in case of new apps needs to be
+  scanned.
+
+### FOSS
+
+During accompanying the QG Checks a couple of things poped up:
```
Comment (hzierer) [inline blog/2023-08-25-OfficeHours.md:25]: I'm afraid "accompanying" is used by default only in a musical context. Maybe assistance or support?

**Review by hzierer — CHANGES_REQUESTED** [review]
on small change request, rest LGTM.

File: blog/2023-08-25-OfficeHours.md (hunk 1 above)
Comment (carslen) [reply blog/2023-08-25-OfficeHours.md:25]: According to Langenscheidt it should be ok, but anyway I'm goint to change it for you :)

File: blog/2023-08-25-OfficeHours.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  scanned.
+
+### FOSS
+
+During the QG Checks a couple of things poped up:
+
+- Commit messages/PR Description  
+  Please do not reference closed systems like Catena-X Consortia Jira Tickets. The content is no public accessible but
+  only for consortia members. Please be descriptive in your Commit messages and Pull Request descriptions.
+- Do not invent new legal documentation (e.g. ``COPYRIGHT.md`), please
```
Comment (Siegfriedk) [inline blog/2023-08-25-OfficeHours.md:30]: is this an oversight/bug with 2 before COPYRIGHT.md and only one closing it?

Comment (carslen) [reply blog/2023-08-25-OfficeHours.md]: Fixed

File: blog/2023-08-25-OfficeHours.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  contains an example how to add a notice section.
+- HasiCorp moved from Mozilla Public License v2.0 (MPL 2.0) to the Business Source License (BSL, or BUSL) v1.1 for
+  future releases of all products and several libraries. Please
+  use [this GH Discussion](https://github.com/eclipse-tractusx/sig-infra/discussions/233) to give feedback if your
+  product has dependencies to HashiCorp products or not.
+
+## Open discussions
+
+- Test automation on environments: Consortia vs Helm test + kind
+- Upgradability of Postgresql Helm Charts
```
[the remark this answers is not in the document]
Comment (carslen) [reply blog/2023-08-25-OfficeHours.md]: added context.

File: blog/2023-08-25-OfficeHours.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  - Contribute to [e2e Helm Chart](https://github.com/eclipse-tractusx/e2e-testing)
+
+### Security
+
+- License for Invicty. Please contact Security Team (Lokesh Gujre or Piotr Stys) in case of new apps needs to be
+  scanned.
+
+### FOSS
+
+During the QG Checks a couple of things poped up:
```
Comment (hzierer) [inline blog/2023-08-25-OfficeHours.md:26]: Better beginning, but totally missed already the first time that it should be "popped" and not "poped" - sorry.

**Review by hzierer — DISMISSED** [review]
LGTM

**Pull request merged**